security/models.py

```diff
@@ -350,12 +351,20 @@ class CeleryTaskLog(SmartModel):
     error_message = models.TextField(verbose_name=_('error message'), null=True, blank=True)
     queue_name = models.CharField(verbose_name=_('queue name'), null=True, blank=True, max_length=250)
     input = models.TextField(blank=True, null=True, editable=False, verbose_name=_('input'))
+    task_args = JSONField(_('task args'), null=True, blank=True)
+    task_kwargs = JSONField(_('task kwargs'), null=True, blank=True)
     output = models.TextField(blank=True, null=True, editable=False, verbose_name=_('output'))
+    retries = models.PositiveSmallIntegerField(verbose_name=_('retries'), null=False, blank=False, default=0)
+    estimated_time_of_arrival = models.DateTimeField(verbose_name=_('estimated time of arrival'), null=False,
+                                                     blank=False)
+    expires = models.DateTimeField(verbose_name=_('time of expiration'), null=True, blank=True)
```

Comment:
time when task will be expired. It can be very helpfull, because task can be stale but still waiting in the queue